packages/opensrp-plans/src/components/NewPlan/tests/index.test.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import { CreatePlanView } from '..';
+import { store } from '@opensrp/store';
+import { createBrowserHistory } from 'history';
+import { Router } from 'react-router';
+import { Provider } from 'react-redux';
+import { mount } from 'enzyme';
+import { Helmet } from 'react-helmet';
+import { act } from 'react-dom/test-utils';
+import { CREATE_PLAN } from '../../../lang';
+
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+const fetch = require('jest-fetch-mock');
+
+const history = createBrowserHistory();
+
+describe('Create Plan Page', () => {
+  afterEach(() => {
+    fetch.resetMocks();
+  });
+
+  it('renders correctly with store(for new product)', async () => {
```

Comment:
After looking at this test I have some questions:

- is this component going to be finally used for EUSM
- if so, do we need tests here that the plan form displayed is what we expect for EUSM?
- what are your plans for doing this different for non-EUSM use cases?

Reply:
hmm,..
. yeah this will be used for EUSM
. will add some test to show that the plan form is as expected,
- I will make a few changes on how props are passed so that the 2 pages create/edit, can be used not just for EUSM. My thought process is to have the 2 as just a wrapper around the form, so the configurations for how the plan-form looks and behaves all resides and are handled by the plan-form. This  means I will not have different useCase for these pages.

Reply:
sounds good